replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/data/ReplicatorConfigImpl.java

```diff
@@ -119,9 +119,10 @@ public void fromMap(Map<String, Object> properties) {
     setSuspended(Boolean.valueOf((String) properties.get(SUSPENDED_KEY)));
     setDeleted(Boolean.valueOf((String) properties.get(DELETED_KEY)));
     setDeleteData(Boolean.valueOf((String) properties.get(DELETE_DATA_KEY)));
-    if (getVersion() >= 2) {
+    if (propVersion >= 2) {
```

Comment:
✏️ I prefer and approach like in `ReplicationSiteimpl` as it make sit easier to maintain and see what changed over time as opposed to performing the logic for handling different versions inside a single method.